ffs/interfaces.go

```diff
@@ -16,7 +16,31 @@ type WalletManager interface {
 	// Balance returns the current balance for an address.
 	Balance(context.Context, string) (uint64, error)
 	// SendFil sends fil from one address to another
-	SendFil(ctx context.Context, from string, to string, amount *big.Int) error
+	SendFil(context.Context, string, string, *big.Int) error
+}
+
+type createVoucherConfig struct {
+	lane uint64
+}
+
+// CreateVoucherOption modifies a createVoucherCOnfig
+type CreateVoucherOption func(*createVoucherConfig)
+
+// WithCreateVoucherLane allows you to specify a non-0 lane
+func WithCreateVoucherLane(lane uint64) CreateVoucherOption {
+	return func(conf *createVoucherConfig) {
+		conf.lane = lane
+	}
+}
+
+// PaychManager provides access to payment channels
+type PaychManager interface {
```

Comment:
We've used a couple different designs with our modules and ffs, but what I chose to do here is just have ffs define the interface and types for `PaychManager`. The implementation's main goal is to support ffs. In fact, I didn't even add rpc, client or cli layers to the importation since that is all accessed through ffs. If we decide sometime there is a need to expose the raw `paych` module as an api, we can, but it would just be extra work for now.